framework/pom.xml

```diff
@@ -204,6 +204,31 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>pl.project13.maven</groupId>
+        <artifactId>git-commit-id-plugin</artifactId>
+        <version>2.2.5</version>
+        <executions>
+          <execution>
+            <id>get-the-git-infos</id>
+            <goals>
+              <goal>revision</goal>
+            </goals>
+          </execution>
+        </executions>
+
+        <configuration>
+          <dotGitDirectory>${project.basedir}/../.git</dotGitDirectory>
+          <generateGitPropertiesFile>true</generateGitPropertiesFile>
+          <generateGitPropertiesFilename>${project.basedir}/../conf/framework.git.properties</generateGitPropertiesFilename>
```

Comment:
I think we can just call this as `git.properties` and keep it in the root level of the repository. That's where it's usually present.

Reply:
There is a problem here. It is conflicting with git.properties from the Drill JDBC driver. And for some reason it was not working when I added it to the root. Maybe because the classpath we set in run_tests: 
`-cp conf/:${DRILL_HOME}/jars/jdbc-driver/drill-jdbc-all-${DRILL_VERSION}.jar:framework/target/framework-1.0.0-SNAPSHOT-shaded.jar`

Of course I can open the properties not as a resource but as a regular file, but I don't like the idea...

Reply:
Do you think swapping the cp order in the run_tests will work? I think we should try and understand what's causing the issue. Using resource is good. Confirming to standard names would also be good.

Reply:
But we would have 2 different git.properties in one classpath, and which one is picked is only a question of order... Is it ok?

Reply:
In Apache projects such as Drill which also have other dependent Apache projects in it - each of which may have git.properties, how do you think it's handled?

Reply:
Well, I'll try to figure it out.

Reply:
Added git.properties to the project root, added the project root to the classpath in run_tests.